### PR TITLE
Add Embedding youtube videos 

### PR DESCRIPTION
**Pull request by LuSilvaDeveloper**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |
| [**179**](https://github.com/Web-Dev-Path/web-dev-path/issues/179) |

#### Have you updated the CHANGELOG.md file? If not, please do it.

Yes.

#### What is this change?

Added Embedding youtube videos.

#### Were there any complications while making this change?

Yes, First I tried to use JSX and styled-components, but after some research, I found that I couldn't use them, so I changed to HTML and CSS files. After that, I found that I couldn't use an external CSS for that purpose, so I had to just put everything inside the HTML file. 

#### How did you verify this change?

The changes will be verified after the merge.

#### When should this be merged?

After review

### CODE REVIEW

**Comment by cherylli** [comment]
Where are we embedding the videos? And how do we view the page with the video?

**Comment by LuSilvaDeveloper** [comment]
According to the issue, it seems that testing the functionality of the embedded videos will be possible only after the pull request (PR) is merged and the changes are deployed and after that, using Liquid code. Liquid is a template language that allows us to display data in a template.

File: CHANGELOG.md [hunk 1]
```diff
@@ -113,7 +113,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Added Google Analytics 4 Integration
-
 ### Fixed
 
 - Lower the file names of GitHub.svg, LikedIn.svg, and YouTube.svg.
+
```
Comment (mariana-caldas) [inline CHANGELOG.md:119]: We only need one "Added" and one "Fixed" per release, @LuSilvaDeveloper . Can you please help us to fix that duplication?

**Review by mariana-caldas — CHANGES_REQUESTED** [review]
Other than the Changelog adjustment, it looks great to me, Lucas. Thanks!

**Review by mariana-caldas — APPROVED** [review]
LGTM! Thanks, @LuSilvaDeveloper !

**Review by cherylli — APPROVED** [review]
looks good

**Review by sergiobtos — APPROVED** [review]
looks good. but correct me if I am wrong, this change only will affect *.md files that project has in the root folder. About the wiki I think will not work as expected. I did some testing with my local projects and I could not see any results with wiki. But anyway, the code is pretty much correct. 

image

**Comment by cherylli** [comment]
> looks good

@sergiobtos, I saw your previous comment which is now edited. Is it working for you now? This came across my mind at first but I did not do any testing.

**Comment by sergiobtos** [comment]
@cherylli 
no, I could not make it work in my code. I tried to implement it in my repo, with no success. Also, from that tutorial, I could not understand how he used _includes. Because the only place that I saw the iframe working was on GitHub pages with this url : https://codepo8.github.io/youtube-embed/demo.html . It shows demo.html, I am confused because it is not  *.md or wiki.

**Comment by cherylli** [comment]
> Hey @cherylli and @sergiobtos !
> 
> Just to give you more context about our Wikipedia. It's a special repository formed by `.md` files that connects directly with GitHub. I encourage you to clone it locally to understand more about its structure.
> 
> In theory :), the `_includes` folder is a convention folder that talks to GitHub when placed on any project root. As @LuSilvaDeveloper mentioned, we will be able to test it properly once the new folder is part of the repository, which means after merging this PR. Feel free to reach out if you have further questions!
> 
> image

but is there a reason why this cannot be tested on a test repo before merging to production? Either way, it's not a critical part that would affect the site so happy for it to be tested after merging

**Pull request merged**